app/scripts/views/mixins/password-mixin.js

```diff
@@ -122,7 +121,7 @@ module.exports = {
   getAffectedPasswordInputs (button) {
     var $passwordEl = this.$(button).siblings('.password');
     if (this.$(button).data('synchronizeShow')) {
-      $passwordEl = this.$('.password');
+      $passwordEl = this.$('.password[data-synchronize-show]');
```

Comment:
This change is needed to prevent the original password from also being displayed when clicking the "show-password" button.